bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/internal/image/SVGRasterizer.java

```diff
@@ -32,4 +32,6 @@ public interface SVGRasterizer {
 	 *         the input is not a valid SVG file or cannot be processed.
 	 */
 	public ImageData rasterizeSVG(InputStream stream, int zoom) throws IOException;
+	
+	public ImageData rasterizeSVG(InputStream stream, int width, int height) throws IOException;
```

Comment:
Please add javadoc here, also if there are any limitations (e.g. what happens when size is negative).